integrations/access/servicenow/types.go

```diff
@@ -52,6 +52,8 @@ type Incident struct {
 	WorkNotes string `json:"work_notes,omitempty"`
 	// Caller is the user on whose behalf the incident is being created. (Must be an existing servicenow user)
 	Caller string `json:"caller_id,omitempty"`
+	// AssignedTo is the person who the incident is assigned.
+	AssignedTo string `json:"assigned_to,omitempty"`
```

Comment:
Should we be more clear which data should populate this field.
From what I understood, it must match a sys_id user from the sys_user table. 
Teleport users can differ from service now users. Is there a good way for matching them?

Reply:
Have clarified the comment to specify that is is a servicenow user name that goes there.
For the caller_id and assigned_to we are assuming that the usernames are the same in servicenow. We make similar assumptions in other plugins at the moment.
I don't think there is a system in place yet for users to configure user mappings for different plugins yet.

Reply:
No problem. As long as we reference that information in the docs, we should be good to go.